dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdTaskUtils.kt

```diff
@@ -0,0 +1,41 @@
+package com.datadog.gradle.plugin
+
+import com.android.builder.model.Version
+import org.gradle.api.Project
+import java.io.File
+
+internal object DdTaskUtils {
+    @Suppress("StringLiteralDuplication")
+    fun resolveDatadogRepositoryFile(target: Project): File {
+        val outputsDir = File(target.buildDir, "outputs")
+        val reportsDir = File(outputsDir, "reports")
+        val datadogDir = File(reportsDir, "datadog")
+        return File(datadogDir, "repository.json")
+    }
+
+    fun findDatadogCiFile(projectDir: File): File? {
+        var currentDir: File? = projectDir
+        var levelsUp = 0
+        while (currentDir != null && levelsUp < DdNdkSymbolFileUploadTask.MAX_DATADOG_CI_FILE_LOOKUP_LEVELS) {
```

Comment:
Probably we need to move `MAX_DATADOG_CI_FILE_LOOKUP_LEVELS` to the parent task at least or even better to have it here, in the utils class?